lib/puppet/provider/cisco_bgp/nxapi.rb

```diff
@@ -200,11 +199,6 @@ def timer_bestpath_limit_set
     @bgp_vrf.timer_bestpath_limit_set(limit, always)
   end
 
-  def confederation_peers_set
-    return unless @property_flush[:confederation_peers]
-    @bgp_vrf.confederation_peers_set(@property_flush[:confederation_peers])
-  end
-
   def flush
     if @property_flush[:ensure] == :absent
       @bgp_vrf.destroy
```

Comment:
I changed the setter in the nu from non-autogen to auto-gen (=()). So that's why the explicit getter/setter is removed.

Can you expand on the first comment - I don't see that in other providers like bgp neighbor for example.